stock_move_change_source_location/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module allows you to change the source location of stock moves from the picking
```

Comment:
@MiquelRForgeFlow Could you explain a little your use case ?

Reply:
In https://github.com/OCA/stock-logistics-workflow/pull/868, you found this module great hehe

A client asked us to be able to change the location where you get the stock in a delivery, not wanting the default one coming from the rule. And as we migrate this client to newer versions, we still need to migrate this module.

Reply:
Ok. I wanted to know because some things have been done in wms if that correspond to your flow.

In some situations, it could be great not to generate all stock moves (mainly the internal ones). Then, at a point of time, release the stock flow.

e.g.: Generate only the OUT > Customer move after sale order confirmation.

I suppose this is the case you want to solve when you are not sure at sale confirmation from where should come the products.

https://github.com/OCA/wms/pull/506